src/Components/Publishing/Sections/Text.tsx

```diff
@@ -75,6 +74,10 @@ export class Text extends Component<Props, State> {
   }
 
   transformNode = (node, index) => {
+    // Dont include relay components unless necessary
+    // To avoid 'regeneratorRuntime' error
+    const { LinkWithTooltip } = require("../ToolTip/LinkWithTooltip")
```

Comment:
This seems fine for now but I think we're going to have to live with the regenerator issue for now, so might be good to account for it in various clients which consume reaction.